bld/CLMBuildNamelist.pm

```diff
@@ -3792,6 +3797,8 @@ sub setup_logic_c_isotope {
       $use_c13_timeseries = $nl->get_value('use_c13_timeseries');
       if ( &value_is_true($use_c13_timeseries) ) {
          if ( defined($stream_fldfilename_atm_c13) ) {
+            &add_logical_to_nl_flags( $nl_flags, $nl, "use_c13_timeseries" );
+            &add_logical_to_nl_flags( $nl_flags, $nl, "use_c13_timeseries" );
```

Comment:
```suggestion
            &add_logical_to_nl_flags( $nl_flags, $nl, "use_c13_timeseries" );
```

Reply:
OR should the second be
`  &add_logical_to_nl_flags( $nl_flags, $nl, "use_c13_bombspike" );`

AND I'm not looking into it, so I will just ask:
Are lines 3833-3834 needed or are they redundant?

Reply:
3833-3834 are added to ensure that these are set in all cases. It's also done here as it's going to be used in the next call. So has to be done in that case.

I could add a comment to say this for clarity.